### ISSUE TITLE
Empty Response

### ISSUE THREAD

**Issue by weaselbass** [calcinai/xero-php#480]
Hi, I am trying to use the private Application example.
I have set the config array with the consumer_key and consumer_secret values from the app page and certificate location.

when running the script all it receives is:

`XeroPHP\Remote\Collection Object ( [_associated_objects:protected] => [storage:ArrayObject:private] => Array ( ) )`

I receive no errors. 
Any help would be appreciated.

**Comment by timacdonald** [comment]
@weaselbass are you saying you the result you pasted above is coming from this line?

https://github.com/calcinai/xero-php/blob/master/examples/private.php#L17

**Comment by weaselbass** [comment]
@timacdonald 
Yes, I'm running this exact script with my public_key/consumer_secret inserted.
Path to the certificate is correct and working as intended as when I modify this it will display an error.

**Comment by timacdonald** [comment]
Awesome to hear ya got it sorted
